packages/venia-concept/src/components/Button/index.js

```diff
@@ -0,0 +1,31 @@
+import { Component, createElement } from 'react';
+import PropTypes from 'prop-types';
+
+import classify from 'src/classify';
+import defaultClasses from './button.css';
+
+class Button extends Component {
+    static propTypes = {
+        classes: PropTypes.shape({
+            content: PropTypes.string,
+            root: PropTypes.string
+        }),
+        type: PropTypes.oneOf(['button', 'reset', 'submit'])
+    };
+
+    static defaultProps = {
+        type: 'button'
```

Comment:
I don't feel super strongly about this. However, for devs or designers that are used to writing  plain HTML, the default `type` being `button` is going to be surprising, since the sets the default as `submit`

Reply:
Same

Reply:
As you've emphasized elsewhere, this is optimal for the most common use case. In Venia and most modern apps, most buttons should have a type of `button`. From MDN:

> The button has no default behavior. It can have client-side scripts associated with the element's events, which are triggered when the events occur.

The cases in which we're using a button to submit a form, without that submission being intercepted by client scripts, will be far more rare. (Ironically this may be one of them.) I think supplying `type="submit"` in exceptional cases is reasonable.